itou/companies/management/commands/_import_siae/siae.py

```diff
@@ -11,7 +11,7 @@
 from itou.companies.models import Company
 
 
-def build_siae(active_siae_keys, row, kind):
+def build_siae(row, kind, *, is_active=True):
```

Comment:
J'explique la raison dans le commit associé :
> Keeping the `is_active` logic in case someone reuse that function in the
future for another purpose.

J'ai repris le commit en faisant de `is_active` un paramètre obligatoire et en expliquant ce qu'il faut dans la docstring.